### PR TITLE
nsc-events-android-1-63-create-event-cards-on-home-page

### PR DESCRIPTION
**Pull request by taylorpapke**
resolves: #63  

Screenshot 2023-10-13 at 4 18 19 PM

Screenshot 2023-10-13 at 4 18 31 PM

### CODE REVIEW

File: app/src/main/java/com/example/nsc_events/model/Event.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package com.example.nsc_events.model
+
+data class Event(
+    val eventTitle: String,
+    val eventDescription: String,
+    /* TODO: change date variable to a Date data type instead of string */
+    val eventDate: String,
+    /* TODO: what other event properties should be added? */
```
Comment (Seiyaroo) [inline app/src/main/java/com/example/nsc_events/model/Event.kt:8]: Perhaps end time? Event Host? So is it student body leadership hosting, or a club, or a schoolwide event? Is the event a paid event or a free event? Just spitballing to see what pops up.

**Review by Seiyaroo — COMMENTED** [review]
We should discuss up front what else is needed on that one thread I started. We don't need everything up front I suppose but it helps to plan / flesh out what we can now and if we dont use it, then we can remove it later.

**Review by kkennedy-gh — DISMISSED** [review]
Looks good to me.

**Comment by Seiyaroo** [comment]
@taylorpapke @theGaryLarson some conflicts have arisen, I don't wanna LGTM until they're resolved. :)

**Comment by taylorpapke** [comment]
> We should discuss up front what else is needed on that one thread I started. We don't need everything up front I suppose but it helps to plan / flesh out what we can now and if we dont use it, then we can remove it later.

Yes @Seiyaroo, I went ahead and added the end time, location and cover photo (flyer). In user story #56, @theGaryLarson mentions to only add these 3 details for the moment. I assume this is to test the waters?

**Comment by taylorpapke** [comment]
Screenshot 2023-10-18 at 1 16 45 PM
Screenshot 2023-10-18 at 1 16 57 PM

**Review by Seiyaroo — APPROVED** [review]
LGTM. We will need to keep in mind about the font things brought up by Brandon in todays meeting and make sure that we note to come back later and edit in if NSC gets back to us with their media kit.

**Comment by theGaryLarson** [comment]
@taylorpapke can you resolve the conflicts in HomePage.kt Once those are done. Also, I noticed some of the same changes here that are in PR #76 did you rebase or make the changes independently in each branch?

**Comment by taylorpapke** [comment]
> @taylorpapke can you resolve the conflicts in HomePage.kt Once those are done. Also, I noticed some of the same changes here that are in PR #76 did you rebase or make the changes independently in each branch?

@theGaryLarson Fixed the conflicts. #76 was a new branch created from this one.

**Pull request closed without merging**